templates/site/about.html

```diff
@@ -100,7 +100,57 @@ <h5>{{ row["title"] }}</h5>
         {% set title = "Acknowledgments" %}
         {% set intro_content = "The Marble climate platform was made possible with funding from the following agencies: " %}
         {% set content %}
-            <p>Acknowledgements are coming soon!</p>
+
+            {% set funder_images = [
+                {"image1": "/images/logos/BCID_Supported_H_RGB_pos.png",
```

Comment:
"image1", "image2", etc. are not descriptive and don't really contribute to the understanding of the code.
If the order is the only thing that matters here, just store these images in a list.

Reply:
Please exclusively use relative paths (ie. don't start the path with a `/`).
This is to ensure that the site works even if the root path contains an additional path prefix. 

For example, right now we have an example site running at `daccs.cs.toronto.edu/example-site`. If paths contain an initial `/` then they will direct to `daccs.cs.toronto.edu/image/logos/...` instead of `daccs.cs.toronto.edu/example-site/images/logos/...`

Reply:
Got rid of "/" at beginning of path